doc/train/finetuning.md

```diff
@@ -45,3 +47,121 @@ To obtain a more simplified script, for example, you can change the {ref}`model
         "fitting_net" : {}
     }
 ```
+
+## PyTorch Implementation {{ pytorch_icon }}
+
+In PyTorch version, leveraging the flexibility offered by the framework and the multi-task training capabilities provided by DPA2 [paper](https://arxiv.org/abs/2312.15492),
+we have introduced an updated, more adaptable approach to fine-tuning. This methodology encompasses two primary variations:
+
+### 1. Single-task fine-tuning
+
+This fine-tuning method is similar to the fine-tune approach supported by TensorFlow.
+It utilizes a standard pre-trained model (`pretrained.pt`, single-task pre-trained) and modifies the energy bias within its fitting net before continuing with training.
+The command for this operation is:
+
+```bash
+$ dp --pt train input.json --finetune pretrained.pt
+```
+
+The model section in input.json can be simplified as follows:
+
+```json
+    "model": {
+        "type_map":     ["O", "H"],
+        "descriptor" :  {},
+        "fitting_net" : {}
+    }
+```
+
+Additionally, within the PyTorch implementation, we also support more general multitask pre-trained models,
+which includes multiple datasets for pre-training. These pre-training datasets share a common descriptor while maintaining their individual fitting nets,
+as detailed in the DPA2 [paper](https://arxiv.org/abs/2312.15492).
+For fine-tuning using this multitask pre-trained model (`multitask_pretrained.pt`),
+one can select a specific branch (e.g., `CHOOSEN_BRANCH`) included in `multitask_pretrained.pt` for fine-tuning with the following command:
+
+```bash
+$ dp --pt train input.json --finetune multitask_pretrained.pt --model-branch CHOOSEN_BRANCH
+```
+
+This command will start fine-tuning based on the pre-trained model's descriptor and the selected branch's fitting net.
+If --model-branch is not set, a randomly initialized fitting net will be used.
```

Comment:
It is a duplicate of Section 5.6.3. We may only maintain one and provide a link from another one.